helm_deploy/monitoring/datahub-networkPolicy.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: allow-prometheus-scraping
+  namespace: ${KUBE_NAMESPACE}
+spec:
+  podSelector:
+    matchLabels:
+      app.kubernetes.io/instance: datahub
```

Comment:
just to check my understanding - does this selector match all the datahub components? Or is this just the GMS?

Reply:
Yes. We deploy 2 pods that export metrics (`frontend` and `gms`), and `instance: datahub` is common to both of them